src/10_functions.py

```diff
@@ -1,12 +1,18 @@
 # Write a function is_even that will return true if the passed-in number is even.
 
 # YOUR CODE HERE
-
+def is_even(n):
+    if int(n)/2 == int(int(n)/2):
+        return True
```

Comment:
Good job. We can clean this up by simply having `is_even` return `n % 2 == 0` which makes the return value a bool